tiledb/sm/array_schema/attribute.cc

```diff
@@ -398,6 +448,25 @@ void Attribute::set_default_fill_value() {
   fill_value_validity_ = 0;
 }
 
+ByteVecValue Attribute::default_fill_value(
```

Comment:
The output of `default_fill_value` should be able to feed easily into `set_fill_value`. Right now that's `tuple<void* value, uint64_t size>`.

The fill value should be its own class, however simple it would be. When you need duplicated arguments running around for a single datum, it means you're missing a class. I'm not sure we need to do it in this PR, but if we won't we need to track it.

Reply:
I would rather not do it in this PR and track it in future